doc/api/esm.md

```diff
@@ -1284,6 +1287,8 @@ _defaultEnv_ is the conditional environment name priority array,
 > 1. If _exports_ is an Object with both a key starting with _"."_ and a key not
 >    starting with _"."_, throw an "Invalid Package Configuration" error.
 > 1. If _exports_ is an Object and all keys of _exports_ start with _"."_, then
+>    1. If _exports_ contains any index property keys, as defined in ECMA-262
```

Comment:
I think this should be move up one line to be outside of the `If _exports_ is an Object and all keys of _exports_ start with _"."_, then` block.

Reply:
this should likely link to https://tc39.es/ecma262/#integer-index

Reply:
Thanks, add both these changes.